content/how-to-contribute/code-style-and-quality-common/index.html

```diff
@@ -472,33 +472,33 @@
   Code Style and Quality Guide — Common Rules
   <a class="anchor" href="#code-style-and-quality-guide--common-rules">#</a>
 </h1>
-<h4 id="preamblehahahugoshortcode56s0hbhb">
+<h4 id="preamblehahahugoshortcode60s0hbhb">
```

Comment:
Do you have an idea why your change is updating these ids?

Reply:
Not really sure 
But I ran the same command in the `asf-site` branch and it generates the similar changes in the IDs (except the number after code. 
Is there another way to test it?
I built the site locally and checked the link
```
-  <a class="anchor" href="#preamblehahahugoshortcode56s0hbhb">#</a>
+  <a class="anchor" href="#preamblehahahugoshortcode57s0hbhb">#</a>
``